version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2015.06.19-1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.0.1"
```

Comment:
My plan was to do a verification deploy at 0.0.1, then bump up to 1.0.0 once I confirm the setup works. :)